tox.ini

```diff
@@ -23,12 +23,16 @@ commands =
 deps = black
 commands = black {posargs} arraylias test
 
-
 [testenv:docs]
+# Editable mode breaks macOS: https://github.com/sphinx-doc/sphinx/issues/10943
+usedevelop = False
 deps =
     -r{toxinidir}/requirements-dev.txt
 commands =
-  sphinx-build -b html -W {posargs} docs/ docs/_build/html
+  sphinx-build -j auto -b html -W {posargs} docs/ docs/_build/html
```

Comment:
I recommend you set this to `--keep-going` right after the `-W` so that a single docs failure doesn't stop the build.

FYI the docs failure looks legit in CI.

Reply:
thanks for this suggestion - this is helpful for seeing all of the possible warnings that will pop up so I can fix them all at once